appcenter-crashes/scripts/postinstall.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const path = require('path');
+
+const projectDirectory = path.resolve(__dirname, '..', '..', '..');
+const mockFileContent = `
```

Comment:
nit - If you move the backtick(<code>`</code>) to the beginning of line 6, then the generated **__mocks__/appcenter-crashes.js** file won't have a empty new line at the beginning of the file.